CONTRIBUTING.md

```diff
@@ -175,6 +175,48 @@ This can happen if you perform development on a different machine, pushed them t
 It is encouraged before submission to rebase your changes on top of master and to remove extraneous commits to improve the commit history.
 However, if multiple developers are doing development in a branch it requires coordination before rewriting history using rebase.
 
+### Creating hotfix releases
+To create and release hotfixes, annotated git tags are used; no release branches are created and maintained. To create a hotfix release, follow the steps below:
+
+#### Maintainer will create a hotfix branch from the desired tag in ni/VireoSDK
+* Goto https://github.com/ni/VireoSDK in your browser, click the "Branch:master" button and select the "Tags" tab in the pop-up that appears.
+* Choose the tag (say v10.1.7) on top of which you want to create a hotfix. The button should now say something like "Tag:v10.1.7".
+* Click on the button again, select 'Branches', and type a new name for the branch (for example, 'hotfix'), hit Enter. This will create a new 'hotfix' branch in ni/VireoSDk from the chosen tag (v10.1.7).
+
+#### Contributor will create a local branch with the desired fixes and do a pull request
+* Synchronize your local master to head.
+```console
+git remote update -p
+git merge --ff-only upstream/master
+git push
```

Comment:
For this section you only need to fetch remote objects to see the new branch, updating master is unnecessary.

```
* Fetch new branches and tags from remote
git remote update -p
```